src/win/core.c

```diff
@@ -41,45 +40,6 @@ static uv_loop_t* default_loop_ptr;
 /* uv_once initialization guards */
 static uv_once_t uv_init_guard_ = UV_ONCE_INIT;
 
-
```

Comment:
now that we are avoiding using the `crt`, it's all dead code. these errors can't happen anymore.